ethereum/contracts/nft/NFTBridge.sol

```diff
@@ -232,20 +235,21 @@ contract NFTBridge is NFTBridgeGovernance {
 
         transfer.tokenID = encoded.toUint256(index);
         index += 32;
-
-        uint8 len_uri = encoded.toUint8(index);
+        
+        // Ignore length due to malformatted payload
         index += 1;
+        transfer.uri = string(encoded.slice(index, encoded.length - index));
```

Comment:
Shoudn't we substract another 34 bytes from the slice length to exclude to and toChain from the uri?